### PR TITLE
Update Chapter 3 region-level statistic exercise

### PR DESCRIPTION
**Pull request by zmbc**
Addresses #594 by doing calculations with poverty level as opposed to median income (which, because it is a median, can't be aggregated in a mathematically meaningful way).

I would appreciate a proofread for clarity. In particular, I am not sure if the word "state's" in the second question is clear about what it's asking (which, if there's any confusion, is the lowest/highest/average value *of state numbers* in the region).

### CODE REVIEW

File: 03-attribute-operations.Rmd [hunk 1]
```diff
@@ -780,9 +780,10 @@ Calculate the change in percentages and map them.
 1. Using `us_states` and `us_states_df` create a new object called `us_states_sel`.
 The new object should have only two variables - `median_income_15` and `geometry`.
 Change the name of the `median_income_15` column to `Income`.
-1. Calculate the change in median income between 2010 and 2015 for each state.
-Bonus: What was the minimum, average and maximum median income in 2015 for each region?
-What is the region with the largest increase of the median income?
+1. Calculate the change in the number of residents living below the poverty level between 2010 and 2015 for each state. (Hint: See ?us_states_df for documentation on the poverty level columns.)
```
Comment (Robinlovelace) [inline 03-attribute-operations.Rmd:783]: Very clear question. Easier for beginners than going straight into percentages. Thank you @zmbc !

File: 03-attribute-operations.Rmd [hunk 2]
```diff
@@ -780,9 +780,10 @@ Calculate the change in percentages and map them.
...
+Bonus: Calculate the change in the *percentage* of residents living below the poverty level in each state.
+1. What was the minimum, average and maximum state's number of people living below the poverty line in 2015 for each region?
```
Comment (Robinlovelace) [inline 03-attribute-operations.Rmd:785]: :+1:

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Looking good to me @zmbc. Please @Nowosad or @jannes-m can you take a final check and merge if happy.

**Comment by Nowosad** [comment]
Awesome work @zmbc! Could you also make PR with new questions and solutions to https://github.com/geocompr/geocompkg/blob/master/vignettes/solutions03.Rmd?

**Comment by Robinlovelace** [comment]
> Awesome work @zmbc! Could you also make PR with new questions and solutions to https://github.com/geocompr/geocompkg/blob/master/vignettes/solutions03.Rmd?

Already in there: https://github.com/geocompr/geocompkg/pull/24/files

**Comment by Nowosad** [comment]
Great!

**Comment by Nowosad** [comment]
@Robinlovelace can we merge it now?

**Comment by Robinlovelace** [comment]
@Nowosad just merged. Was waiting for approval from you or @jannes-m. Need to sort our workflow for that. My suggestion: explicit approval on PRs by all of us unless it's minor/urgent or if the PR has been left hanging for more than a few days.

**Pull request merged**